### ISSUE TITLE
Collective to collective donations between hosts

### ISSUE THREAD

**Issue by znarf** [opencollective/opencollective#2441]
## Purpose

We give the ability to collective to give to other collectives from their collective balance. This is a nice, useful and used feature.

However, this is not available for donations between collectives that are on different hosts.

There was a proof of concept using credit cards as a backend, this was deactivated because of various issues in the implementation and because of the payment processor fees. 

## Roadmap & Timeframe

- Fall 2019: scope the problem and outline potential implementations
- Later: implementation (not prioritized)

## Success metrics

- Less support requests
- Number of collective to collective donations

## Link to related past issues

- https://github.com/opencollective/opencollective/issues/1196
- https://github.com/opencollective/opencollective/issues/1247
- https://github.com/opencollective/opencollective/issues/1238

**Comment by alanna** [comment]
Also #2017

**Comment by xdamman** [comment]
As a first step, it would be great to allow such donations for admins of the host of the collective giving the money.

The [CCC collective](https://opencollective.com/ccc) made 3x a $5k donation to [/RebelsManager](https://opencollective.com/RebelsManager]. I had to make a collective to collective donation across two different hosts (c3 -> c6). It created the `Order` with an `ERROR` status. So I had to manually edit the status and set it to `PENDING`.
